src/app/pages/OriginsLaunchpad/pages/Dashboard/components/PreviousSalesCardSection/index.module.scss

```
@@ -0,0 +1,6 @@
+.myntImage {
+  @apply tw-mb-0 tw-flex tw-flex-col tw-justify-center;
```

The image should change in size. The myntImage class is a sad fit, it should have never be implemented with the wrong image... :face_exhaling: 
```suggestion
  @apply tw-flex tw-flex-col tw-justify-center tw-flex-none tw-mb-0 ;
```

---

Well, both Mynt and Fish images are a bit different than the rest of the images. The cleanest solution would be to ask @t-z-u to prepare both of them in the exact same format and use new images because now we have to hack borders and dimensions and it just doesn't look as good as it would if it was unified.

---

I added it to the Fix Image Formats Task
https://sovryn.monday.com/boards/2218344956/pulses/2235657942

---

I already wrote to tzu so if he could prepare it, I'll change it in this task, otherwise it will be taken care of in Fix Image Formats Task.